### PR TITLE
Change world to cubes, fix shader issue

### PR DESCRIPTION
**Pull request by rgarmsen2295**
I'm going to change how that shader looks/is structured but at least it's working right (I think, looks good at least).

Also switched to a mini (3 cube) town for cookies to hit soon

### CODE REVIEW

File: resources/phong_vert.glsl [hunk 1]
```diff
@@ -7,21 +7,14 @@ uniform mat4 P;
...
 void main() {
 	worldPos = (V * M * vertPos).xyz;
-	lightVert = (V * vec4(lightPos, 1.0)).xyz;
-
-	lightVert = normalize(lightVert - worldPos);
 
 	vec3 vertNormal = normalize(vertNor);
-
-	fragNor = normalize((V * M * vec4(vertNormal, 0.0)).xyz);
+	fragNor = normalize((transpose(inverse(V * M)) * vec4(vertNormal, 0.0)).xyz);
```
Comment (nurgan) [inline resources/phong_vert.glsl:17]: transpose inverse should be computed on CPU side, but doesn't make a difference for now

**Review by nurgan — APPROVED** [review]
Looks good. Left a comment for one thing, that can be fixed some other time.

File: resources/phong_vert.glsl (hunk 1 above)
Comment (rgarmsen2295) [reply resources/phong_vert.glsl:17]: Yeah I agree, just a quick fix for now, made issue #32 to track

File: src/main.cpp [hunk 2]
```diff
@@ -44,8 +44,7 @@ std::shared_ptr<Shape> shapeGirl;
 
 // Main directional light properties
 // TODO(rgarmsen2295): Move into shader manager class
-//Light curLight = {-1.75, 10, -1.75, 1, 1, 1, 0, 1};
-Light curLight = { 0, 10, 0, 1, 1, 1, 0, 1 };
+Light curLight = { -10.0f, 10.0f, 0.0f, 1.0f, 1.0f, 1.0f, 0.0f, 1.0f };
```
Comment (joncatanio) [inline src/main.cpp]: This line is breaking my build. Looks like the `Light` struct has a GLint as it's last member, but here it is assigned a float.

Comment (rgarmsen2295) [reply src/main.cpp]: OSX! Yeah that makes sense, fixing...

**Review by joncatanio — APPROVED** [review]
The GLint thing was my ownly concern, looks like you got it figured out.

**Pull request merged**